lib/complex/kernel.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Complex.Kernel do
+  @moduledoc """
+  Provides operator overloading for Elixir operators.
```

Comment:
We need to mention that if you use Complex.Kernel, the arithmetic operators are no longer supported in guards.

Reply:
I've added the paragraph below for that

Reply:
I've also updated the README to reflect Complex.Kernel